expertise/execute_expertise.py

```diff
@@ -0,0 +1,177 @@
+from pathlib import Path
+import openreview
+from .create_dataset import OpenReviewExpertise
+from .dataset import ArchivesDataset, SubmissionsDataset, BidsDataset
+from .config import ModelConfig
+
+# Move run.py functionality to a function that accepts a config dict
+def execute_expertise(config_file=None, model_config=None):
```

Comment:
what is the difference between config_file and model_config?

Reply:
The difference is `model_config` is already a ModelConfig object whereas `config_file` would be a dictionary that still needs to be turned into a ModelConfig object.
The different use cases are seen here, where the command line expects a JSON file, and uses the `config_file_path` argument:
https://github.com/openreview/openreview-expertise/blob/a86bddea5e48d9efd20d00f8fe1a0f1f1be7e0f8/expertise/run.py#L10-L13
whereas the server directly reads a JSON/dictionary from the POST request:
https://github.com/openreview/openreview-expertise/blob/a86bddea5e48d9efd20d00f8fe1a0f1f1be7e0f8/expertise/service/routes.py#L37
which later gets turned into a ModelConfig using the `config_dict` argument:
https://github.com/openreview/openreview-expertise/blob/a86bddea5e48d9efd20d00f8fe1a0f1f1be7e0f8/expertise/execute_expertise.py#L12-L13

Reply:
I would define `execute_expertise( model_config=None)` where model_config can be created from a JSON that comes in the request body or from file.